sparql11/data-sparql11/functions/data5.ttl

```diff
@@ -0,0 +1,6 @@
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <http://example.org/> .
+
+# string data with characters outside the basic multilingual plane
+:s8 :str "\U0001F46A"^^xsd:string . # U+1F46A FAMILY
```

Comment:
No need for `^^xsd:string` - the tests should work as RDF 1.0 as simple literals.

Reply:
Will fix. This was inadvertently included as a copy-paste from data.ttl.